src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/wrapped/WrappedBlacklistAssignmentRequestManager.java

```diff
@@ -15,9 +15,9 @@
 {
 
     @NotNull
-    private final Collection<IToken> blackListedResolvers;
+    private final Collection<IToken<?>> blackListedResolvers;
 
-    public WrappedBlacklistAssignmentRequestManager(@NotNull final IStandardRequestManager wrappedManager, @NotNull final Collection<IToken> blackListedResolvers)
+    public WrappedBlacklistAssignmentRequestManager(@NotNull final IStandardRequestManager wrappedManager, @NotNull final Collection<IToken<?>> blackListedResolvers)
     {
         super(wrappedManager);
         this.blackListedResolvers = blackListedResolvers;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "blackListedResolvers". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)